app/src/main/java/kr/co/connect/boostcamp/livewhere/ui/detail/DetailActivity.kt

```diff
@@ -0,0 +1,162 @@
+package kr.co.connect.boostcamp.livewhere.ui.detail
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import kr.co.connect.boostcamp.livewhere.R
+import kr.co.connect.boostcamp.livewhere.databinding.ActivityDetailBinding
+import kr.co.connect.boostcamp.livewhere.model.MarkerInfo
+import kr.co.connect.boostcamp.livewhere.ui.map.StreetMapActivity
+import kr.co.connect.boostcamp.livewhere.util.generateUuid
+import kr.co.connect.boostcamp.livewhere.util.keyboardHide
+import kr.co.connect.boostcamp.livewhere.util.keyboardShow
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+
+class DetailActivity : AppCompatActivity() {
+
+    companion object {
+        private const val DETAIL_CONTAINER_ID = R.id.detail_activity_container
+    }
+
+    private val viewModel: DetailViewModel by viewModel()
+    private lateinit var binding: ActivityDetailBinding
+    private lateinit var currentFragment: Fragment
+    lateinit var markerInfo: MarkerInfo
```

Comment:
```suggestion
    private lateinit var markerInfo: MarkerInfo
```
P1. DetailActivity에서만 쓰인다면 private를 꼭 붙여주세요